scripts/lint.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
-
 set -ex # fail on first error, print commands
 
+echo "Checking code style with black...."
+python -m black bambi --line-length=100  --target-version=py37 --exclude=tests/ --check
```

Comment:
Getting there! I would suggest this for this line, as the parameters are coming from the toml file now, and we do want checks on the tests. I would add the py37 option to the toml file as well

```suggestion
python -m black bambi --check
```

Reply:
Yea, I was having trouble adding the target version to the toml file, so I left it out.. 

```
Error: Could not open file /Users/camen/Documents/bambi/pyproject.toml: Error reading configuration file: invalid literal for int() with base 0: 'py37' (line 3 column 1 char 31)
```

re: changing `lint.sh`. I thought about changing it, but thought that it might be good to leave the entire declaration as its acting as self-documentation (instead of relying on the user to know that they have to look at pyproject.toml for what black is actually running)

re: exclude=tests/ - yea that's my b, forgot to remove...

Do you have any thoughts on the first 2 points?

Reply:
This is the `pyproject.toml` file that produces that error

```
[tool.black]
line-length = 100
target-version = py37
```

I tried removing the space, putting in quotes, just having 37, and reading black's documentation https://black.readthedocs.io/en/stable/pyproject_toml.html

Reply:
hahaha seconds later I have found it.... 

https://github.com/psf/black/blob/master/pyproject.toml

Needs to be a list apparently

Reply:
So now the only thing to resolve is: what do you think about leaving the invocation in `scripts/lint.sh` as is? I also noticed that arviz also includes the parameters defined in the pyproject.toml (but that could just be an oversight)

Reply:
ah that is an oversight. For using lint.sh or using it directly whatever you prefer. I have no opinion

Reply:
Sounds good! Thanks for the review :)

Reply:
279999d